docs/consumer.adoc

```diff
@@ -63,7 +63,7 @@ Jenkins then notifies you and the developer of the other feature branch that the
 
 In order to allow for some rollback of some of these side-effects, typically one would:
 
-* Create a job that just builds the master branch.
+* Create a job that just builds the controller branch.
```

Comment:
This "master" word seems to be about git branches, so "controller" does not works here, if I remember correctly the "main branch" can be used.

```suggestion
* Create a job that just builds the main branch.
```

Reply:
I agree with the suggestion from @aHenryJard .  The "main" word is preferred here.